### DISCUSSION TITLE
Shortcut for Bookmarks?

### DISCUSSION THREAD

**Discussion by dm4tsuo** [mucommander/mucommander#713] [Ideas]
Is it possible to add shortcuts for added bookmarks? For instance, I put 3 bookmarks: home, dir1 and dir2. And I could access them as ctrl+1, ctrl+2 and ctrl+3. This would improve productivity a lot. Is it possible? Is it interesting?

**Comment by ahadas** [comment]
@dm4tsuo nope, it's not possible to define shortcuts for specific bookmarks. There's the bookmark quick-list (defaults to ALT+4) that enable easier selection of bookmarks but not as simple as a shortcut per bookmark.

About the concrete shortcuts you suggested (CTRL+1, CTRL+2, ...) - note that these are used for switching between windows. I think that use case of having multiple windows is less relevant now that we have multiple tabs and can execute operations in the background, but it's still there
